pages/badge-single.jsx

```diff
@@ -0,0 +1,217 @@
+    HeroUnit = require('../components/hero-unit.jsx'),
+    SocialShare = require('../components/social-share.jsx'),
+    BadgeHorizontalIcon = require('../components/badge-horizontal-icon.jsx'),
+    RequirementsList = require('../components/requirement-list.jsx'),
+    Badge = require('../components/badge.jsx'),
+    Link = require('react-router').Link,
+    $ = require('jquery'),
+    _ = require('underscore');
+
+/**
+ * Badges Navigation component
+ */
+var SingleBadgePageNavigation = React.createClass({
+    propTypes : {
+        nextNavigationDetails : function () {
+            return React.PropTypes.object({
+                url: React.PropTypes.string.isRequired,
+                icon: React.PropTypes.string.isRequired,
+                icon2x: React.PropTypes.string
+            });
+        },
+        prevNavigationDetails : function () {
+           return React.PropTypes.object({
+               url : React.PropTypes.string.isRequired,
+               icon : React.PropTypes.string.isRequired,
+               icon2x : React.PropTypes.string
+           });
+        }
+    },
+    render : function () {
+        return (
+            <div className="single-badge-page-navigation">
+                <div className="row">
+                    <div className="col-xs-6">
+                        <Link to={this.props.prevNavigationDetails.url} className="prev">
+                            <div className="icon">
+                                <Badge
+                                    title="Previous"
+                                    icon={this.props.prevNavigationDetails.icon}
+                                    icon2x={this.props.prevNavigationDetails.icon2x} />
+                            </div>
+                            <div className="text"><span className="fa fa-long-arrow-left"></span> Previous</div>
+                        </Link>
+                    </div>
+                    <div className="col-xs-6 text-right">
+                        <Link to={this.props.nextNavigationDetails.url} className="next">
+                            <div className="text">Next <span className="fa fa-long-arrow-right"></span></div>
+                            <div className="icon">
+                                <Badge
+                                    title="Next"
+                                    icon={this.props.nextNavigationDetails.icon}
+                                    icon2x={this.props.nextNavigationDetails.icon2x} />
+                            </div>
+                        </Link>
+                    </div>
+                </div>
+            </div>
+        );
+    }
+});
+
+/**
+ * Single badge Page
+ * should take a badgeId
+ */
+var BadgePage = React.createClass({
+    statics: {
+        pageTitle: 'Badges',
+        pageClassName: 'badges single-badge'
+    },
+    onQualificationsSubmit : function ( event ){
+        event.preventDefault();
+        console.log( $( event.target ).serializeArray() );
```

Comment:
if we remove jQuery, a `console.log(event.nativeEvent.target)` should work for debugging purposes